proto/import_sstpb.proto

```diff
@@ -283,6 +283,7 @@ message Pair {
 message WriteBatch {
     uint64 commit_ts = 1;
     repeated Pair pairs = 2;
+    uint64 start_ts = 3;
```

Comment:
Would you add some comments about this field? I guess the `start_ts` is more like a hint that needed to be provided while uploading a SST in write CF.

Reply:
https://github.com/pingcap/kvproto/pull/1224/commits/e0ee9d117f8f88ada2816320e817b9c6a79398d7

Reply:
(Also I guess such field should also be stored in `SstMeta` somehow. Because the observer can only get information about the SST in `SstMeta`.)

Reply:
I can move start_ts into SstMeta, do you think it's enough? just to avoid the two places have different values.

Reply:
https://github.com/pingcap/kvproto/pull/1224/commits/1421347a6a24c53bdc85c6552cb8233bc95ccd77